docs/basics/quickstart.rst

```diff
@@ -74,11 +74,11 @@ We can now see the streams if we run the `list` command from the tool.
 Step 5: Start a write proxy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now, lets start a write proxy server that serves writes to distributedlog namespace `distributedlog://127.0.0.1/messaging/distributedlog`. The server listens on 8000 to accept fan-in write requests.
+Now, lets start a write proxy server that serves writes to distributedlog namespace `distributedlog://127.0.0.1/messaging/my_namespace`. The server listens on 8000 to accept fan-in write requests.
 
 ::
     
-    > ./distributedlog-service/bin/dlog com.twitter.distributedlog.service.DistributedLogServerApp -p 8000 --shard-id 1 -sp 8001 -u distributedlog://127.0.0.1:7000/messaging/distributedlog -mx -c ${DL_HOME}/distributedlog-service/conf/distributedlog_proxy.conf
+    > ./distributedlog-service/bin/dlog-daemon.sh start writeproxy -p 8000 --shard-id 1 -sp 8001 -u distributedlog://127.0.0.1:7000/messaging/my_namespace -mx -c `pwd`/distributedlog-service/conf/distributedlog_proxy.conf
```

Comment:
@sijie as the offline discussion, a wrapper is preferred to `c.t.dl.service.DistributedLogServerApp` directly, I find this already exists.